apps/docs-snippets/src/guide/provider/provider.test.ts

```diff
@@ -21,7 +21,7 @@ describe('Provider', () => {
     expect(provider).toBeInstanceOf(Provider);
     expect(consensusParameters).toBeDefined();
     expect(consensusParameters).toBeInstanceOf(Object);
-  })
+  });
```

Comment:
Why wasn't this caught by the linter?

Reply:
Because `prettier` wasn't being run in `apps/docs-snippets` and my editor auto-formatted it. I created a separate PR for it in #2038. We can track this discussion there.